cfme/tests/distributed/test_appliance_replication.py

```diff
@@ -114,59 +92,25 @@ def test_appliance_replicate_between_regions(request, virtualcenter_provider):
         initialEstimate: 1/4h
         casecomponent: Appliance
     """
-    appl1, appl2 = get_replication_appliances()
-
-    def finalize():
-        appl1.destroy()
-        appl2.destroy()
-    request.addfinalizer(finalize)
-    appl1.ipapp.browser_steal = True
-    with appl1.ipapp:
-        configure_db_replication(appl2.hostname)
-        virtualcenter_provider.create()
-        appl1.collections.infra_providers.wait_for_a_provider()
-
-    appl2.ipapp.browser_steal = True
-    with appl2.ipapp:
-        appl2.collections.infra_providers.wait_for_a_provider()
-        assert virtualcenter_provider.exists
-
-
-@pytest.mark.tier(2)
-@pytest.mark.ignore_stream("upstream")
-def test_external_database_appliance(request, virtualcenter_provider, appliance):
-    """Tests that one appliance can externally
-       connect to the database of another appliance.
-
-    Metadata:
-        test_flag: replication
+    configure_replication_appliances(temp_appliances_unconfig_funcscope_rhevm)
+    remote_app, global_app = temp_appliances_unconfig_funcscope_rhevm
 
-    Polarion:
-        assignee: tpapaioa
-        initialEstimate: 1/4h
-        casecomponent: Appliance
-    """
-    appl1, appl2 = get_distributed_appliances(appliance)
-
-    def finalize():
-        appl1.destroy()
-        appl2.destroy()
-    request.addfinalizer(finalize)
-    appl1.ipapp.browser_steal = True
-    with appl1.ipapp:
+    remote_app.browser_steal = True
+    with remote_app:
         virtualcenter_provider.create()
-        appl1.collections.infra_providers.wait_for_a_provider()
+        remote_app.collections.infra_providers.wait_for_a_provider()
 
-    appl2.ipapp.browser_steal = True
-    with appl2.ipapp:
-        appl2.collections.infra_providers.wait_for_a_provider()
+    global_app.browser_steal = True
+    with global_app:
+        global_app.collections.infra_providers.wait_for_a_provider()
         assert virtualcenter_provider.exists
 
 
 @pytest.mark.tier(2)
 @pytest.mark.ignore_stream("upstream")
-def test_appliance_replicate_sync_role_change(request, virtualcenter_provider, appliance):
-    """Tests that a role change is replicated
+def test_external_database_appliance(
```

Comment:
The test steps here look like they're identical to the test case above, what am I missing?

Reply:
Of course I see it after sending that comment. Different method calls in the first step.